### PR TITLE
docs: update readme of outlier filter (#758)

### PR DESCRIPTION
**Pull request by wep21**
* add outlier filter readme template

* wip

* wip

* replace image to drawio.svg

* delete doc of occupancy_grid_map_filter in outlier_filter

* update voxel_grid_outlier_filter

* remove occupancy grid map outlier filter svg

* update readme

* fix ring outlier filter image

* add dual_return_outlier_filter readme

* fix readme

* update dual-return-outlier-filter readme

Co-authored-by: yukke42 <muramatsuyusuke9542@gmail.com>

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [ ] Read [commit-guidelines][commit-guidelines]
- [ ] Assign PR to reviewer

If you are adding new package following items are required:

- [ ] Documentation with description of the package is available
- [ ] A sample launch file and parameter file are available if the package contains executable nodes

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets

## CI Checks

- **Build and test for PR / build-and-test-pr**: Required to pass before the merge.
- **Build and test for PR / clang-tidy-pr**: NOT required to pass before the merge. It is up to the reviewer(s). Found false positives? See the [guidelines][clang-tidy-guidelines].
- **Check spelling**: NOT required to pass before the merge. It is up to the reviewer(s). See [here][spell-check-dict] if you want to add some words to the spell check dictionary.

[commit-guidelines]: https://www.conventionalcommits.org/en/v1.0.0/
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

File: sensing/pointcloud_preprocessor/docs/dual-return-outlier-filter.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+## Inputs / Outputs
+
+This implementation inherits `pointcloud_preprocessor::Filter` class, please refer [README](../README.md).
+
+### Output
+
+| Name                                           | Type                                       | Description                                             |
+| ---------------------------------------------- | ------------------------------------------ | ------------------------------------------------------- |
+| `/dual_return_outlier_filter/frequency_image`  | `sensor_msgs::msg::Image`                  | The histogram image that represent visibility           |
+| `/dual_return_outlier_filter/visibility`       | `autoware_debug_msgs::msg::Float32Stamped` | A representation of visibility with a value from 0 to 1 |
```
Comment (tkimura4) [inline sensing/pointcloud_preprocessor/docs/dual-return-outlier-filter.md:31]: autoware_debug_msgs -> tier4_debug_msgs ?

Comment (wep21) [inline sensing/pointcloud_preprocessor/docs/dual-return-outlier-filter.md:31]:
```suggestion
| `/dual_return_outlier_filter/visibility`       | `tier4_debug_msgs::msg::Float32Stamped` | A representation of visibility with a value from 0 to 1 |
```

Comment (wep21) [reply sensing/pointcloud_preprocessor/docs/dual-return-outlier-filter.md]: fixed at https://github.com/autowarefoundation/autoware.universe/pull/208/commits/d44cd104602f03ff800f6bfe5818836c12fb6e12

**Review by tkimura4 — APPROVED** [review]
LGTM

**Pull request merged**